### PR TITLE
Add CHANGELOG for 0.16

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

**Comment by phadej** [comment]
Really? first changelog without any typos, unbelievable!

File: servant/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,72 @@
...
+
+    ```diff
+    -FailureResponse Response
+    +-- | The server returned an error response including the
+    +-- failing request. 'requestPath' includes the 'BaseUrl' and the
+    +-- path of the request.
+    +FailureResponse (RequestF () (BaseUrl, BS.ByteString)) Response
+    ```
+
+- *servant-client* Fix (impelement) `StreamBody`instance
```
Comment (domenkozar) [inline servant/CHANGELOG.md]: typo :D

File: servant/CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,72 @@
 [The latest version of this document is on GitHub.](https://github.com/haskell-servant/servant/blob/master/servant/CHANGELOG.md)
 
-next
+0.16
 ----
 
+### Signifacant changes
```
Comment (arianvp) [inline servant/CHANGELOG.md]:
```suggestion
### Significant changes
```

Comment (arianvp) [reply servant/CHANGELOG.md]: Typo :D

**Comment by alpmestan** [comment]
I was too tired to spot them apparently :-) Thanks Domen & Arian!

**Pull request merged**